File: feb07/src/add_server.py
#!/usr/bin/env python3
#coding:utf-8

import rospy
from feb07.srv import *

def server():
    rospy.init_node("add_server")
    s = rospy.Service("add_two_number_py",test,handle_function)
    rospy.loginfo("waiting to request")
    rospy.spin()

def handle_function(req):
    rospy.loginfo("two number are %lf and %lf" , req.a, req.b)
    return testResponse(req.a+req.b)

if __name__=="__main__":
    server();

File: feb07/src/add_server.cpp
#include <ros/ros.h>
#include <feb07/test.h>

bool add_callback(feb07::test::Request &req,feb07::test::Response &res)
{
    res.c = req.a + req.b;
    std::cout<<"The server responsed\n";
    return true;
}

int main(int argc, char **argv)
{
    ros::init(argc,argv,"add_server");
    ros::NodeHandle nh;
    ros::ServiceServer server = nh.advertiseService("add_two_number",add_callback);
    while(ros::ok()){
        std::cout<<"Waiting to add\n";
        ros::spin();
    }
    return 0;
}